### DISCUSSION TITLE
GitHub organization view doesn't show private content

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#133149] [Mobile] [answered]
GitHub Mobile beta for Android 

I'm a member of an organization. The organization has publicly and privately pinned repositories. 

When I browse to the organization, I see the publicly pinned repositories. 

Expected results: as a member, unless I actively select a mode to show the public view, I should see the member 📌📌📌📌📌 s.

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

That is an excellent suggestion ❤️  Created an internal issue to track this feature request 🙇 

Cheers!

**Reply by nrgbistro** [reply]
Similarly, on GitHub mobile for IOS, I can't see any private content that I'm a member of. It's as if I'm not signed into my private organization at all. This would make it much easier to respond to PR comments on the go!
